extensions/adobe/experience/customerJourneyManagement/message-experienceevent.schema.json

```diff
@@ -0,0 +1,106 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/message-experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent Template Mixin",
+  "type": "object",
+  "description": "Adobe CJM ExperienceEvent mixin for ingesting the ExperienceEvent. Includes the core/standard ExperienceEvent as well as the required Adobe CJM ExperienceEvent mixins.",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "definitions": {
+    "messageExperienceevent": {
+      "properties": {
+        "xdm:messageExecution": {
+          "title": "Message Execution",
+          "$ref": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageexecution",
+          "description": "MessageExecution details for the Adobe CJM ExperienceEvent."
+        },
+        "xdm:messageProfile": {
```

Comment:
The message Profile is part of a message Execution only, should should that be made part of thaty only?